integration-test/src/commonTest/kotlin/sample/BasicTypesSerializationTest.kt

```diff
@@ -138,7 +138,7 @@ class BasicTypesSerializationTest {
             val name = inp.nextUntil(':', '}')
             if (name.isEmpty())
                 return READ_DONE
-            val index = desc.getElementIndexOrThrow(name)
+            val index = desc.getElementIndex(name)
```

Comment:
why not `orThrow`? at least add manual `check(index != UNKNOWN_NAME)`

Reply:
Because that's a test and it will fail anyway, was following the path of the least resistance